Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.4
+
+RUN mkdir -p /opt/cheppers/meeting-room-dasboard
+WORKDIR /opt/cheppers/meeting-room-dasboard
+COPY . .
+
+RUN apk add --no-cache g++ make supervisor ruby ruby-dev ruby-io-console ruby-json ca-certificates && \
+    echo "gem: --no-rdoc --no-ri" > ~/.gemrc && \
+    gem install bundler && \
+    bundler --without development
+
+EXPOSE 80
+CMD ["bundle exec puma", "config.ru", "-p80"]
```

Comment:
Bundle exec could be an entrypoint. And cmd would be puma config.ru... In that case in run if you want to debug something you can simply say `docker run whatever irb`. Which would come up with an irb overriding cmd by default but still using entrypoint to setup the environment. This is just a nice to have.